native/src/main/java/io/ballerina/persist/cmd/Generate.java

```diff
@@ -0,0 +1,252 @@
+ *  "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ *  KIND, either express or implied.  See the License for the
+ *  specific language governing permissions and limitations
+ *  under the License.
+ */
+package io.ballerina.persist.cmd;
+
+import io.ballerina.cli.BLauncherCmd;
+import io.ballerina.compiler.syntax.tree.SyntaxTree;
+import io.ballerina.persist.PersistToolsConstants;
+import io.ballerina.persist.nodegenerator.BalSyntaxTreeGenerator;
+import io.ballerina.persist.objects.BalException;
+import io.ballerina.persist.objects.Entity;
+import io.ballerina.projects.DiagnosticResult;
+import io.ballerina.projects.Package;
+import io.ballerina.projects.PackageCompilation;
+import io.ballerina.projects.Project;
+import io.ballerina.projects.ProjectEnvironmentBuilder;
+import io.ballerina.projects.ProjectException;
+import io.ballerina.projects.directory.BuildProject;
+import io.ballerina.projects.directory.ProjectLoader;
+import io.ballerina.tools.text.TextDocument;
+import io.ballerina.tools.text.TextDocuments;
+import org.ballerinalang.formatter.core.Formatter;
+import org.ballerinalang.formatter.core.FormatterException;
+import picocli.CommandLine;
+
+import java.io.File;
+import java.io.IOException;
+import java.io.PrintStream;
+import java.io.PrintWriter;
+import java.nio.charset.StandardCharsets;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+import java.util.regex.Pattern;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
+
+
+/**
+ * Class to implement "persist" command for ballerina.
+ */
+@CommandLine.Command(
+        name = "generate",
+        description = "generate database configurations."
+        )
+
+public class Generate implements BLauncherCmd {
+
+    private static final PrintStream errStream = System.err;
+
+    private String sourcePath = "";
+    public ProjectEnvironmentBuilder projectEnvironmentBuilder;
+
+    private static final String COMMAND_IDENTIFIER = "persist-generate";
+
+    private CommandLine parentCmdParser;
+
+    Project balProject;
+
+    String name;
+
+    @CommandLine.Option(names = {"-h", "--help"}, hidden = true)
+    private boolean helpFlag;
+
+    @Override
+    public void execute() {
+        if (helpFlag) {
+            String commandUsageInfo = BLauncherCmd.getCommandUsageInfo(COMMAND_IDENTIFIER);
+            errStream.println(commandUsageInfo);
+            return;
+        }
+        try  {
+            if (projectEnvironmentBuilder == null) {
+                balProject = ProjectLoader.loadProject(Paths.get(""));
+            } else {
+                balProject = ProjectLoader.loadProject(Paths.get(this.sourcePath), projectEnvironmentBuilder);
+            }
+            name = balProject.currentPackage().descriptor().org().value() + "." + balProject.currentPackage()
+                    .descriptor().name().value();
+
+
```

Comment:
```suggestion
```

Reply:
Updated in [895edde](https://github.com/ballerina-platform/persist-tools/pull/15/commits/895eddeb28babc700ff8c472fe1745b73ebe9286)